DuckDuckGo/SpecialErrorPage/SpecialErrorPageInterfaces.swift

```diff
@@ -0,0 +1,131 @@
+//
+//  SpecialErrorPageInterfaces.swift
```

Comment:
I don't think we should put all these protocols under this umbrella file - for me personally it makes things confusing - I'm fine with putting separate files into group though named like this

Reply:
Sounds good to me!